sketch_rnn.py

```diff
@@ -28,415 +30,445 @@
 
 ###################################### hyperparameters
 class HParams():
-    def __init__(self):
-        self.data_location = 'cat.npz'
-        self.enc_hidden_size = 256
-        self.dec_hidden_size = 512
-        self.Nz = 128
-        self.M = 20
-        self.dropout = 0.1
-        self.batch_size = 100
-        self.eta_min = 0.01
-        self.R = 0.99995
-        self.KL_min = 0.2
-        self.wKL = 0.5
-        self.lr = 0.001
-        self.lr_decay = 0.9999
-        self.min_lr = 0.00001
-        self.grad_clip = 1.
-        self.temperature = 0.4
-        self.max_seq_length = 200
+	def __init__(self):
```

Comment:
Users can change the hyperparameters as and when they want. Is there a need to add `BIGNET`?

Reply:
It is unnecessarily increasing the code size.